tracing/src/main/java/com/palantir/remoting3/tracing/AsyncSlf4jSpanObserver.java

```diff
@@ -192,11 +50,4 @@ public static AsyncSlf4jSpanObserver of(
             String serviceName, InetAddress ip, Logger logger, ExecutorService executorService) {
         return new AsyncSlf4jSpanObserver(serviceName, ip, logger, executorService);
     }
-
-    @Override
-    public void doConsume(Span span) {
```

Comment:
this was an implementation detail of the abstract `AsyncSpanObserver` class, so I think it's ok to delete.